src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -1,14 +1,30 @@
 @extends "php/common.snip"
 
-@snippet decorateSampleCode(apiMethod, coreSampleCode)
+@snippet decorateSampleCode(xapiClass, apiMethod, coreSampleCode)
     try {
-        ${@apiMethod.apiVariableName} = new {@apiMethod.apiClassName}();
+        @if xapiClass.hasMissingDefaultOptions
+            $options = [
+                @if xapiClass.missingDefaultServiceAddress
+                    'serviceAddress' => 'my-service-address',
+                @end
+                @if xapiClass.missingDefaultServiceScopes
+                    'scopes' => ['my-service-scope'],
```

Comment:
I'm not sure I know what you mean - my-service-address and my-service-scope are placeholders in the method sample, which the user has to set to the appropriate values. Maybe they should be in [MY-SERVICE-ADDRESS] format?